jujugui/static/gui/src/app/components/deployment-flow/direct-deploy/test-direct-deploy.js

```diff
@@ -22,6 +24,17 @@ describe('DirectDeploy', function() {
       renderMarkdown={options.renderMarkdown || sinon.stub()} />
   );
 
+  beforeEach(() => {
+    windowJujulib = window.jujulib;
+    window.jujulib = {
+      URL: urls.URL
+    };
+  });
+
+  afterEach(() => {
+    window.jujulib = windowJujulib;
+  });
+
```

Comment:
jujulib no longer exists on the window in the tests. Once the jujulib modules are require()d we can remove this setup.